### PR TITLE
♻️ Redirecting `inputs retrieval` via `dynamic-scheduler` ⚠️

### PR DESCRIPTION
**Pull request by GitHK**
## ⚠️ devops
- `dynamic-scheduler` also uses existing `DYNAMIC_SIDECAR_API_SAVE_RESTORE_STATE_TIMEOUT`
- no action required

## What do these changes do?

Inputs retrieval is now redirected via `dynamic-scheduler`.

Bonus:
- unified retrieve calls in webserver

## Related issue/s

- fixes https://github.com/ITISFoundation/osparc-simcore/issues/3191
- https://github.com/ITISFoundation/osparc-simcore/issues/5632
- https://github.com/ITISFoundation/osparc-simcore/issues/6894
- closes https://github.com/ITISFoundation/osparc-simcore/issues/6909

## Dev-ops checklist

- [x] No ENV changes or I properly updated ENV ([read the instruction](https://git.speag.com/oSparc/osparc-ops-deployment-configuration/-/blob/configs/README.md?ref_type=heads#how-to-update-env-variables))

### CODE REVIEW

**Comment by YuryHrytsuk** [comment]
There is a new ENV. 

Shall we expect the releated PR in config repo?

**Comment by GitHK** [comment]
> There is a new ENV.
> 
> Shall we expect the releated PR in config repo?

@YuryHrytsuk just updated the description. it's and existing env. I expect no changes.

**Review by YuryHrytsuk — APPROVED** [review]

File: services/dynamic-scheduler/tests/unit/api_rpc/test_api_rpc__services.py [hunk 1]
```diff
@@ -55,14 +58,14 @@ def node_not_found(faker: Faker) -> NodeID:
 @pytest.fixture
 def service_status_new_style() -> DynamicServiceGet:
     return TypeAdapter(DynamicServiceGet).validate_python(
-        DynamicServiceGet.model_config["json_schema_extra"]["examples"][1]
+        DynamicServiceGet.model_json_schema()["examples"][1]
```
Comment (sanderegg) [inline services/dynamic-scheduler/tests/unit/api_rpc/test_api_rpc__services.py:61]: nice, good to know

**Review by sanderegg — APPROVED** [review]

**Review by pcrespov — COMMENTED** [review]
thx. But I have the same question as in https://github.com/ITISFoundation/osparc-simcore/pull/6943#pullrequestreview-2496106386

**Review by pcrespov — APPROVED** [review]
thx for the explanation.

**Pull request merged**